modules/administration-guide/examples/snip_che-editor-id-list.adoc

```diff
@@ -0,0 +1,9 @@
+:_content-type: SNIPPET
```

Comment:
feel free to keep this snippet as is but unless you plan to reuse it in other docs, you can simply add the instructions to the concealing editors procedure

Reply:
This snippet has already been used in several documents: `Configuring default editor` and `Concealing editors`. Let's leave it as is.